public/RateLimitError.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Rate Limit Error</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <style>
```

Comment:
Maybe this is a little too picky, but could we put the styles in a separate file? Just to separate our concerns (content from styling) a bit more? Thanks!

Reply:
Possibly, let me check. The pages in the public folder a little bit different then our standard ruby on rails pages in the app directory. I modeled these pages after the default 404, 422, and 500 error pages which include the style tag in those html pages.

Reply:
All right, to be consistent I suppose it's all right if we leave those that way.